src/DataTableAbstract.php

```diff
@@ -779,7 +779,7 @@ protected function searchPanesSearch(): void
     /**
      * Count filtered items.
      */
-    protected function filteredCount(): int
+    public function filteredCount(): int
```

Comment:
I modified the visibility of that method to public because the `totalCount()` method is also public.